src/ovs/aca_ovs_programmer.cpp

```diff
@@ -31,6 +32,48 @@ extern bool g_demo_mode;
 
 namespace aca_ovs_programmer
 {
+static int aca_set_port_vlan_workitem(const string port_name, uint vlan_id)
+{
+  ACA_LOG_DEBUG("aca_set_port_vlan_workitem ---> Entering\n");
+
+  ulong not_care_culminative_time = 0;
+  int overall_rc;
+
+  if (port_name.empty()) {
+    throw std::invalid_argument("port_name is empty");
+  }
+
+  if (vlan_id == 0) {
+    throw std::invalid_argument("vlan_id is zero");
+  }
+
+  static ushort MAX_PORT_WAIT_SECONDS = 300; // 5 mins
```

Comment:
I know this may be a short-term fix but let us still make it as a configuration, at least a constant in an easy-read conf properties file.

Reply:
added to new files aca_config.h

Reply:
Awesome and thank you.